webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/RudderDit.scala

```diff
@@ -309,6 +325,40 @@ class RudderDit(val BASE_DN:DN) extends AbstractDit {
     }
   }
 
+  object RULETARGET extends CATEGORY(
+      uuid = "rootRuleCategory"
+    , parentDN = BASE_DN
+    , name = "Rules"
+    , description = "This is the main category of Rules"
+    , isSystem = true
+    , objectClass = OC_SPECIAL_TARGET
+    , objectClassUuid = A_RULE_CATEGORY_UUID
+  ) {
+    ruleTargetServer =>
+
```

Comment:
I don't think it's a category, just a new entry type. 
I think it should go under `SYSTEM` and use `SYSTEM` dn in its `ruleTargetDN`